sparrow-py/pysrc/sparrow_py/__init__.py

```diff
@@ -1,7 +1,6 @@
 """Kaskada query builder and local executon engine."""
 from .expr import Expr
-from .session import Table
-from .session import Session
+from .session import Session, Table, InvalidSchemaException
```

Comment:
The linter is going to expand these imports again.

Reply:
Run `nox` (no arguments) and make sure all sessions pass. We don't have anything in CI yet.